Sources/core/objects/Projectiles.cpp~

```diff
@@ -0,0 +1,321 @@
+#include "Projectiles.h"
```

Comment:
Yes I said this in the other thread, but I always had to remove them manually(I mean remove them then commit and push) so that will help me a lot.